### PR TITLE
Eb sprint 5 responsiveness

### PR DESCRIPTION
**Pull request by ericberglund117**
## Description

- Refactor the color scheme to be more accessible. Only with certain font colors
- Add media queries. Specifically for the header/search input, pantry display, and modify-pantry
- Clean up search input scss styling
- Refactor HTML search form to be more accessible 

## Motivation and Context

- Better accessibility for the user

## How Has This Been Tested?

- Through browser testing. 
- Lighthouse 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [] My change requires a change to the documentation.
- [] I have updated the documentation accordingly.
- [] I have added tests to cover my changes.
- [] All new and existing tests passed.

### CODE REVIEW

File: src/css/styles.scss [hunk 1]
```diff
@@ -24,29 +24,29 @@ h1 {
...
   cursor: pointer;
   height: 17%;
   margin: 5px 5px 10px 0px;
   transition: .8s ease;
-};
+}
 
 h4 {
-  color: $recipe-font-color;
+  color: #092000;
```
Comment (Kristiannmiller) [inline src/css/styles.scss:43]: Love this color change. Much easier to see with higher contrast between text and button color.

File: src/css/styles.scss [hunk 2]
```diff
@@ -80,54 +80,55 @@ header {
...
-  }
+}
+#search-input {
+  background-color: #b6dcca;
+  background: -webkit-gradient( linear,left top, left bottom, color-stop(0, rgb(255, 255, 255)), color-stop(1, rgb(225,225,225)));
+  border-radius: 10px 0px 0px 10px;
+  border: none;
+  box-shadow: inset -3px -1px 20px 1px #5c7894;
+  color: $blue-background-color;
+  font-family: $default-body-font-family, sans-serif;
```
Comment (Kristiannmiller) [inline src/css/styles.scss:91]: This change to the font family was a great call. Calibri is way more Dyslexia friendly

File: src/css/variables.scss [hunk 3]
```diff
@@ -1,6 +1,5 @@
 $blue-background-color: #5B7894;
-$default-body-font-family: 'Quicksand';
-$recipe-type-color: #84C8A4;
+$default-body-font-family: 'calibri';
```
Comment (chadrick-d-dev) [inline src/css/variables.scss:2]: I think calibri was a great choice for accessibility font.

File: src/css/styles.scss [hunk 4]
```diff
@@ -532,3 +534,62 @@ main {
...
+  }
+}
+
+@media only screen and (max-width: 900px) {
+  .drop-menu {
+    margin-left: 71%;
+  }
+}
+
+@media only screen and (max-width: 725px) {
```
Comment (Kristiannmiller) [inline src/css/styles.scss:568]: Love that sliding pantry with the media queries! Looks great!

File: src/index.html [hunk 5]
```diff
@@ -11,11 +11,11 @@
   <body>
     <header>
       <h1><span>fresh</span>Picks<img src="./images/apple-logo.png" alt="apple icon" class="header-apple-icon"></h1>
-      <nav id="search" role="search">
-        <input id="search-input" type="text" autocomplete="off" placeholder="Search Recipes..."></input>
-        <label for="search-input" class="search-label">Search</label>
+      <form id="search" role="search">
```
Comment (chadrick-d-dev) [inline src/index.html:14]: Great catch everyone. Good to remember that search bars and buttons belong with form and label tags

**Comment by Kristiannmiller** [comment]
Looks great! Let's do one more refactor, get the readme up to date, and get this turned in!

File: src/css/styles.scss (hunk 1 above)
Comment (chadrick-d-dev) [reply src/css/styles.scss:43]: I also feel it was a great improvement from the previous color.

**Review by chadrick-d-dev — APPROVED** [review]

**Pull request merged**